src/components/addTask_form/addTask_form.html

```diff
@@ -0,0 +1,35 @@
+<div class="wrapper">
+  <form action="" class="form">
+    <section class="description">
+      <fieldset class="description">
+        <label for="description">Description </label>
+        <input class="enter" id="description" type="text" value="Enter Task" />
+      </fieldset>
```

Comment:
Fieldsets are only for a group of radio buttons I think.
```suggestion
        <label for="description">Description </label>
        <input class="enter" id="description" type="text" value="Enter Task" />
```